doc/UsersGuide/source/solving.rst

```diff
@@ -158,6 +158,142 @@ with :ref:`SUNDIALS/IDA <sundials_ida>` integrator and with enabled
+    Placement(visible = true, transformation(origin = {10, 48}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Mechanics.MultiBody.Joints.Revolute revolute2 annotation(
+    Placement(visible = true, transformation(origin = {10, -10}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+equation
+  connect(prismatic.frame_b, fixed2.frame_b) annotation(
+    Line(points = {{40, -60}, {60, -60}, {60, -60}, {60, -60}}, color = {95, 95, 95}));
+  connect(fixed1.frame_b, revolutePlanar.frame_a) annotation(
+    Line(points = {{-70, 70}, {-60, 70}, {-60, 70}, {-60, 70}}));
+  connect(revolutePlanar.frame_b, fixedTranslation1.frame_a) annotation(
+    Line(points = {{-40, 70}, {-20, 70}, {-20, 70}, {-20, 70}}, color = {95, 95, 95}));
+  connect(fixedTranslation1.frame_b, revolute1.frame_a) annotation(
+    Line(points = {{0, 70}, {10, 70}, {10, 58}, {10, 58}}, color = {95, 95, 95}));
+  connect(revolute1.frame_b, fixedTranslation2.frame_a) annotation(
+    Line(points = {{10, 38}, {10, 38}, {10, 30}, {10, 30}}, color = {95, 95, 95}));
+  connect(revolute2.frame_b, prismatic.frame_a) annotation(
+    Line(points = {{10, -20}, {10, -20}, {10, -60}, {20, -60}, {20, -60}}));
+  connect(revolute2.frame_b, body2.frame_a) annotation(
+    Line(points = {{10, -20}, {10, -20}, {10, -30}, {20, -30}, {20, -30}}, color = {95, 95, 95}));
+  connect(revolute2.frame_a, fixedTranslation2.frame_b) annotation(
+    Line(points = {{10, 0}, {10, 0}, {10, 10}, {10, 10}}, color = {95, 95, 95}));
+  connect(fixedTranslation1.frame_b, body1.frame_a) annotation(
+    Line(points = {{0, 70}, {18, 70}, {18, 70}, {20, 70}}));
+end piston;
+
+.. omc-mos ::
+
+  setCommandLineOptions("-d=initialization");
+  buildModel(piston);
+
+Note how OpenModelica will inform the user about relevant and irrelevant start
+values for this model and for which variables a fixed default start value is
+assumed.
+The model has four joints but only one degree of freedom, so one of the joints
+`revolutePlanar` or `prismatic` must be initialized.
+
+
+So, initializing `phi` and `w` of `revolutePlanar` will give a sensible start
+system.
+
+.. omc-loadString ::
+
+  model pistonInitialize
+    extends piston(revolute1.phi.fixed = true, revolute1.phi.start = -1.221730476396031, revolute1.w.fixed = true, revolute1.w.start = 5);
+  equation
+  end pistonInitialize;
+
+.. omc-mos ::
+
+  setCommandLineOptions("-d=initialization");
+  simulate(pistonInitialize, stopTime=2.0);
+
+
+.. omc-gnuplot :: piston
+  :caption: Vertical movement of mass body2.
+
+  body2.frame_a.r_0[1]
+
+Homotopy Method
+~~~~~~~~~~~~~~~
+
+For complex start conditions OpenModelica can have trouble finding a solution
+for the initialization problem with the default newton method.
+
+Modelica offers the homotopy operator [#f5]_ to formulate a actual and
+simplified expression for equations. OpenModelica has differen solvers
+available for non-linear systems. If the homotopy operator is used inside the
```

Comment:
I don't get this. If the homotopy operator is _not_ used, then it doesn't matter what homotopyOnFirstTry is set to. I'd rather say that if the homotopy operator is used and homotopyOnFirstTry is set (which is the default case), then...

Reply:
Why not just state 

Initializing with homotopy on the first try is default if a homotopy operator is used. It can be switched off with `--homotopyOnFirstTry=false`.

Reply:
KISS :)

Or, if you prefer, use Ockham's razor

Reply:
It is currently `--noHomotpyOnFirstTry` to disable it. I might change that to `homotopyOnFirstTry=false` at some point.